guide/using-the-thymio-ii-robot.md

```diff
@@ -80,27 +100,18 @@ To open the Thymio II window, just double click on the 3D model of the robot.
 
 %end
 
-### Thymio II Pen
+#### thymio2.wbt
 
-You can easily add a pen in the simulated model by adding a `Thymio2Pen` node to the `Thymio2::bodySlot` field.
-This pen can draw on every 3D object.
-This is particularily useful to perform [these exercises](https://aseba.wikidot.com/en:thymiodrawing).
-An example of this is located there (see the [figure](#simulation-of-a-thymio-ii-having-a-pen)): "WEBOTS\_HOME/projects/robots/thymio/worlds/thymio2\_pen.wbt".
+![thymio2_example.png](images/thymio2_example.png) An example of the possible behaviors of the Thymio II.
 
-%figure "Simulation of a Thymio II having a pen."
+#### thymio2\_pen.wbt
 
-![thymio2_pen.png](images/thymio2_pen.png)
+![thymio2_pen.png](images/thymio2_pen.png) You can easily add a pen in the simulated model by adding a `Thymio2Pen` node to the `Thymio2::bodySlot` field.
+This pen can draw on every 3D object.
+This is particularly useful to perform [these exercises](https://aseba.wikidot.com/en:thymiodrawing).
 
-%end
 
-### Thymio II Ball
+#### thymio2\_ball.wbt
 
-[These exercises](https://aseba.wikidot.com/en:thymioballeinverse) can be reproduced by adding a `Thymio2Ball` node in your world.
+![thymio2_ball.png](images/thymio2_ball.png) [These exercises](https://aseba.wikidot.com/en:thymioballeinverse) can be reproduced by adding a `Thymio2Ball` node in your world.
```

Comment:
This screenshot is not readable: we can hardly understand what/where is the robot there. I would drop the robot window on these images and zoom a little bit on the robot so that the robot is more visible and the screenshot more understandable.